CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # go-ipfs changelog
 
+### 0.4.9 - 2017-04-30
+
+- Features
```

Comment:
Those below aren't all features. Do you want to separate them into categories or to just change title to Changes?